Sources/KlaviyoCore/Utils/FileUtils.swift

```diff
@@ -69,13 +69,22 @@ public func removeFile(at url: URL) -> Bool {
     return false
 }
 
-/// load any plist from app main bundle
+/// load any plist from app main bundle or React Native framework bundle
 /// - Parameter name: the name of the plist
 /// - Returns: the contents of the plist in `[String: AnyObject]` or nil if not found
 func loadPlist(named name: String) -> [String: AnyObject]? {
+    // Try loading from main bundle first
     if let path = Bundle.main.path(forResource: name, ofType: "plist"),
        let dict = NSDictionary(contentsOfFile: path) as? [String: AnyObject] {
         return dict
     }
+
+    // If not found in main bundle, try loading from React Native framework bundle
+    if let reactNativeBundle = Bundle(identifier: "org.cocoapods.klaviyo-react-native-sdk"),
+       let path = reactNativeBundle.path(forResource: name, ofType: "plist"),
+       let dict = NSDictionary(contentsOfFile: path) as? [String: AnyObject] {
+        return dict
+    }
+
```

Comment:
Oh gross, but good find!

Reply:
This might mean we'll need a similar solution for the flutter SDK. I don't remember how far we got with helping them set this. @dan-peluso do you have any idea? Quick google tells me SPM is now encouraged for flutter integration anyway, but cocoapods is still supported.

Reply:
We didn't get too far! But I'm not sure if we tried this method!

Reply:
yea but their bundle identifier won't be klaivyo-react-native-sdk

Reply:
yeah I just learnt from the other thread that we are using the bundle of the cocoapod and not the app. Wondering if there is a general solution for this @ab1470 if not I think what we have now is good and we can revisit when we want this to work for flutter.